### PR TITLE
Update set-server-disks.md

### PR DESCRIPTION
**Pull request by danielgainer**
Just adding quotation marks around the other file sizes.

### CODE REVIEW

File: Servers/set-server-disks.md [hunk 1]
```diff
@@ -57,17 +57,17 @@ Use this API operation when you want to change the disks on an existing server.
           "value":[
              {
                "diskId":"0:0",
-               "sizeGB":1
+               "sizeGB":"1"
```
Comment (srveit) [inline Servers/set-server-disks.md]: Add a comma at end to make it valid JSON

File: Servers/set-server-disks.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ Use this API operation when you want to change the disks on an existing server.
...
 			   "type":"raw"
              },
              {
                "diskId":"0:1",
-               "sizeGB":2
+               "sizeGB":"2"
```
Comment (srveit) [inline Servers/set-server-disks.md]: Add a comma at end to make it valid JSON

File: Servers/set-server-disks.md [hunk 3]
```diff
@@ -57,17 +57,17 @@ Use this API operation when you want to change the disks on an existing server.
...
 			   "type":"raw"
              },
              {
                "diskId":"0:2",
-               "sizeGB":16
+               "sizeGB":"16"
```
Comment (srveit) [inline Servers/set-server-disks.md]: Add a comma at end to make it valid JSON

**Comment by jim-phillips** [comment]
@danielgainer Please fix the code per comments.

**Comment by srveit** [comment]
Closing as it is now fixed.

File: Servers/set-server-disks.md (hunk 3 above)
Comment (Sammy13782) [reply Servers/set-server-disks.md]: ,

**Pull request closed without merging**